bin/pbf

```diff
@@ -102,10 +81,14 @@ function compileMessages(parent, parentName, enums, read, write) {
     }
 }
 
-function repeatedDest(fields) {
+function repeatedDest(fields, enums) {
     var repeated = [];
     for (var i = 0; i < fields.length; i++) {
-        if (fields[i].repeated && !fields[i].options.packed) repeated.push('"' + fields[i].name + '": []');
+        var field = fields[i];
+        if (field.repeated && !field.options.packed)
+            repeated.push(field.name + ': []');
+        else if (enums[field.type] && field.options.default !== undefined)
+            repeated.push(field.name + ': ' + JSON.stringify(field.options.default));
```

Comment:
Well, the code assumed this even before by using `field.name` as-is e.g. in https://github.com/mapbox/pbf/pull/40/files#diff-40802e6c310a24cc1b1508f6d219d5f5L56.

And the spec has following definition, so I don't see why not:
```
ident = letter { letter | unicodeDigit | "_" }
```